pom.xml

```diff
@@ -31,7 +31,7 @@
 		<org.bidib.org.oxbow.version>1.2.2</org.bidib.org.oxbow.version>
 		<net.coderazzi.version>5.5.1</net.coderazzi.version>
 		<com.sun.mail.version>1.6.2</com.sun.mail.version>
-		<org.projectlombok.version>1.18.12</org.projectlombok.version>
```

Comment:
Please remove it as it's a local change. 🗑️

Reply:
I did it.